app/controllers/barcode_items_controller.rb

```diff
@@ -2,7 +2,7 @@
 # anomaly here is the :find action, which has some special logic built-in to it, see the comments below.
 class BarcodeItemsController < ApplicationController
   def index
-    @items = Item.gather_items(current_organization, @global)
```

Comment:
I can't find anywhere where this `@global` instance variable is set, so this change should keep the query the same.